draft-kazuho-httpbis-priority.md

```diff
@@ -442,10 +442,11 @@ server and the intermediary.
 
 Once a client sends a request, circumstances might change and mean that it is
 beneficial to change the priority of the response. As an example, a web browser
-might issue a prefetch request for an HTML document with the urgency parameter
+might issue a prefetch request for a JavaScript file with the urgency parameter
 of the Priority request header field set to `background`. Then, when the user
-navigates to the HTML while prefetch is in action, it would send a
-reprioritization frame with the priority field value set to `urgency=0`.
+navigates to a page which references the new JavaScript file, while the prefetch
```

Comment:
Assuming that the example talks about SPA, is there a "page" navigation?

I do not mind merging the PR as-is, and I do not know about SPA, but I am just wondering if there is a technical error in the example.

Reply:
I agree, but a) this works for a normal page navigation as well (you can prefetch files for the next full page load) and b) in the context of an SPA, we typically still speak of a "navigation" even though it doesn't include a full page load (e.g., the URI is updated, browser history is appended, etc.). 

I felt it was easier to just keep "navigate" than to start talking about the SPA case in particular.

Reply:
Thank you for the explanation. I think I'm fine with the text.